pkg/domain/infra/abi/play.go

```diff
@@ -1343,6 +1343,21 @@ func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, body io.Reader, opt
 				return nil, fmt.Errorf("unable to read YAML as Kube Pod: %w", err)
 			}
 			podNames = append(podNames, podYAML.ObjectMeta.Name)
+
+			for _, vol := range podYAML.Spec.Volumes {
+				switch vs := vol.VolumeSource; {
+				case vs.PersistentVolumeClaim != nil:
+					volumeNames = append(volumeNames, vs.PersistentVolumeClaim.ClaimName)
+				case vs.ConfigMap != nil:
+					volumeNames = append(volumeNames, vs.ConfigMap.Name)
+				case vs.HostPath != nil:
+					volumeNames = append(volumeNames, vs.HostPath.Path)
```

Comment:
Why is this trying to delete host paths? I don't think we are creating named volumes for that so it shouldn't try to delete them?!

Reply:
You're right, my bad.